README.rst

```diff
@@ -20,6 +20,10 @@ Symbols:
 
 ``pip install python-language-server``
 
+You can then start the server with this command:
+
+``python -m pyls``
```

Comment:
Does just running `pyls` not work for you?

Reply:
Doesn't seem like it. I install it without `sudo` so it probably shouldn't in my case and in general

Reply:
Did you make sure `pyls` is in your path?

Reply:
No i didn't, because i didn't know where it was located. Now i see that it's in `~/.local/bin/pyls` on my system and it's not in my `$PATH`.
I propose this change because it looks like this method (`python -m pyls`) is less ambiguous and error-prone. May be i'm wrong so please feel free to correct me

Reply:
Less error-prone is right. I don't have ~/.local/bin/pyls on my computer. but python -m pyls works. pip show -v python-language-server enabled me to guess that it was in /usr/local/bin, which *is* in my path.

But I didn't know that command. And had to alias it to "python-language-server" for my editor plugin to find it.